docs/source/user_guide/input_output.md

```diff
@@ -176,7 +176,12 @@ ds = load_bboxes.from_numpy(
 
 The resulting data structure `ds` will include the centroid trajectories for each tracked bounding box, the boxes' widths and heights, and their associated confidence values if provided.
 
-For more information on the bounding boxes data structure, see the [movement bounding boxes dataset](target-poses-and-bboxes-dataset) page.
+:::{note}
```

Comment:
Since this note is only relevant for the VIA tracks file, not for `from_numpy`, how about putting it under the corresponding tab.

Otherwise some people may wonder if `from_numpy` also expects the top-left corners.